.github/workflows/pipeline.yml

```diff
@@ -1,20 +1,38 @@
 name: Pipeline
 
 on:
-  push:
-    branches: [main]
```

Comment:
Why not run this on a push to main?

Reply:
We can but I didn't see the point - we know the tests will pass and how the coverage will look from the results of the PR that the push to main came from, right? I can put it back just didn't really see the point